### PR TITLE
Add Snowbridge initialize migration on Westend

### PR DESCRIPTION
**Pull request by yrong**
# Description

Fix https://github.com/paritytech/polkadot-sdk/pull/5074 which missed the runtime migration to initialize channels of the bridge.

### CODE REVIEW

**Comment by claravanstaden** [comment]
This is not really a backport, so perhaps change the title to "Add Snowbridge initialize migration on Westend", otherwise the title is misleading.

File: cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/lib.rs [hunk 1]
```diff
@@ -209,7 +216,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("bridge-hub-westend"),
 	impl_name: create_runtime_str!("bridge-hub-westend"),
 	authoring_version: 1,
-	spec_version: 1_016_000,
+	spec_version: 1_017_000,
```
Comment (acatangiu) [inline cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/lib.rs:219]:
```suggestion
	spec_version: 1_016_001,
```

**Review by acatangiu — APPROVED** [review]

**Review by bkontur — APPROVED** [review]

**Comment by yrong** [comment]
https://github.com/paritytech/polkadot-sdk/actions/runs/10923847845/job/30321556177?pr=5747

`check-runtime-migration(bridge-hub-rococo)` fails though seems nothing to do with this PR. 

@acatangiu Is there anything required left for this PR?

**Comment by acatangiu** [comment]
@yrong just some spurious network issues leading to some CI tests failing, restarted those CI jobs, should be auto-merged when they finish

**Review by alistair-singh — APPROVED** [review]

**Review by serban300 — APPROVED** [review]

**Pull request merged**